src/patterns/structural/small-crates.md

```diff
@@ -26,10 +26,35 @@ fine-grained dependencies.
   has both versions 1.0 and 0.5. Since the `Url` from `url:1.0` and the `Url`
   from `url:0.5` are different types, an HTTP client that uses `url:0.5` would
   not accept `Url` values from a web scraper that uses `url:1.0`.
-- Packages on crates.io are not curated. A crate may be poorly written, have
+  Rust and Cargo provide several options to handle this disadvantage:
+  - [workspace.dependencies](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table)
+    helps in enforcing the same crate version across all workspace members
+  - dependnecy can be renamed in cargo.toml and in Rust module import 
```

Comment:
```suggestion
  - a dependency can be renamed in `Cargo.toml` and in Rust module import 
```
anyway, how renaming a dependency help with dependency hell?

Reply:
why mentioning dependency hell if it does not exist?